docs/operations-guide/running-the-metabase-jar-file.md

```diff
@@ -1,49 +1,36 @@
-# Running the Metabase Jar File
+# Running the Metabase JAR file
 
-To run the Metabase jar file you need to have Java installed on your system. Currently Metabase requires Java 8 or higher and will work on either the OpenJDK or Oracle JRE.
+To run Metabase via a JAR file, you will need to have a Java Runtime Environment (JRE) installed on your system.
 
-### Download Metabase
-
-If you haven't done so already the first thing you need to do is [Download Metabase](https://metabase.com/start/jar.html).  Simply save the .jar file to a folder on your system where you wish to run Metabase.
-
-
-### Verify Java is installed
-
-Before you can launch the application you must verify that you have Java installed.  To check that you have a working java runtime, go to a terminal and type:
+### Install Java JRE
 
-    java -version
+We recommend the latest LTS version of JRE from [AdoptOpenJDK](https://adoptopenjdk.net/releases.html) with HotSpot JVM and x64 architecture, but other [Java versions](./java-versions.md) are supported too.
 
-You should see output such as:
-
-    java version "1.8.0_31"
-    Java(TM) SE Runtime Environment (build 1.8.0_31-b13)
-    Java HotSpot(TM) 64-Bit Server VM (build 25.31-b07, mixed mode)
-
-If you did not see the output above and instead saw either an error or your Java version is less than 1.8, then you need to install the Java Runtime (JRE).
-
-[OpenJDK Downloads](http://openjdk.java.net/install/)
-[Oracle's Java Downloads](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+### Download Metabase
 
+Go to the [Metabase Download Page](https://metabase.com/start/jar.html) and download the latest release. Place the downloaded JAR-file into a newly created directory (as it will create some files when it is run)
```

Comment:
```suggestion
Go to the [Metabase download page](https://metabase.com/start/jar.html) and download the latest release. Place the downloaded JAR file into a newly created directory (as it will create some files when it is run).
```